sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.aest.controller.ts

```diff
@@ -94,7 +94,7 @@ export class EducationProgramAESTController extends BaseController {
     @Body() payload: ApproveProgramAPIInDTO,
   ): Promise<void> {
     await this.programService.approveEducationProgram(
-      new Date(payload.effectiveEndDate),
+      payload.effectiveEndDate,
```

Comment:
shouldn't this be a a date object 
![image](https://user-images.githubusercontent.com/77353155/198697204-7aa30e93-e265-41d1-8363-bbcd72221d10.png)

Reply:
effectiveEndDate is a string in entity model as it is date only column now.

Reply:
pls, ignore the comment. i saw the change